frontend/src/Admin/containers/ShowSubmissionsTab.js

```diff
@@ -17,5 +18,18 @@ export default compose(
           }
         })
     })
+  }),
+  graphql(FinalizeShowInvites, {
+    props: ({ mutate }) => ({
+      finalizeInvites: id =>
```

Comment:
Instead of passing around the `id` all over the place, you could probably get it from react-router in `ownProps`